parameters/world_modeling/world_model.yaml

```diff
@@ -70,29 +70,24 @@ plugins:
       odom_model:
           map_frame: map
           odom_frame: odom
-          base_link_frame: base_footprint_wheel
+          base_link_frame: base_footprint
           alpha1: 0.05   # rot -> trans + strafe
           alpha2: 0.5    # trans -> rot
           alpha3: 0.5    # trans -> trans
           alpha4: 0.5    # rot -> rot
           alpha5: 0      # trans -> strafe
 
 #- name: laser_integration
-#  lib: libed_laser_plugin.so
-#  frequency: 40
-#  parameters:
-#      laser_topic: /hero/base_laser/scan
-#      min_segment_size_pixels: 10
-#      world_association_distance: 0.4
-#      segment_depth_threshold: 0.2
-#      min_cluster_size: 0.2
-#      max_cluster_size: 1.0
-#      max_gap_size: 10
-#- name: entity_clearer
-#  lib: libed_clearer_plugin.so
-#  enabled: 1
-#  parameters:
-#      entity_timeout: 1.0
+  #lib: libed_laser_plugin.so
```

Comment:
Do we want to keep the laser plugin or can we remove it entirely?

Reply:
Yep that will be fixed in https://github.com/tue-robotics/hero_bringup/pull/17 which will be merged into this branch. When finished.